rocketpool/api/node/status.go

```diff
@@ -233,7 +234,10 @@ func getStatus(c *cli.Context) (*api.NodeStatusResponse, error) {
 	wg.Go(func() error {
 		alerts, err := alerting.FetchAlerts(cfg)
 		if err != nil {
-			return err
+			// no reason to make `rocketpool node status` fail if we can't get alerts
+			// (this is more likely to happen in native mode than docker where
+			// alertmanager is more complex to set up)
+			alerts = make([]*models.GettableAlert, 0)
```

Comment:
Suggest printing a little warning or something here so the user knows that there was an issue getting alerts, just a `printf` without erroring out would be fine; otherwise it looks like there aren't any, which may not be strictly true. I submit that the lack of an alert manager when one is enabled is alert-worthy by itself.

Reply:
Agree it is worthy to make user aware. printf is a bit tough here since it is in the api container. I added a Warning to the struct so we can marshal the warning message back to the user but keep `rocketpool node status` resilient in the face of misconfigured/broken alertmanager. Done in e95fafcb